### ISSUE TITLE
A11y: a keyboard user cannot upload a new file

### ISSUE THREAD

**Issue by danielszenasi** [box/box-ui-elements#2579]
Issue : a keyboard cannot upload a new file

Steps to reproduce the issue:
1. Try the example available in box documentation https://opensource.box.com/box-ui-elements/#/Elements/ContentUploader
2. Now try to reach the Browse your device link with the keyboard
